randovania/games/prime1/game_data.py

```diff
@@ -96,9 +96,9 @@ def _exporter():
             "NTSC 0-00, meaning that all versions of Prime are guaranteed to be logically completable when randomized.",
         ),
         (
-            "What rooms in Magmoor Caverns not superheated?",
+            "Why do I not take heat damage in some rooms in Magmoor, despite not having Varia Suit?",
             "Elevator rooms, Storage Cavern, and every room between Twin Fires and the Phazon Mines and "
-            "Phendrana Drifts elevators (Far Magmoor), Burning Trail, and Lake Access.",
+            "Phendrana Drifts elevators (Far Magmoor), Burning Trail, and Lake Access are not coded to be superheated.",
```

Comment:
I'd rephrase the answer here a little. Something like "It is vanilla behaviour, that only some rooms are superheated. Here is a list of them: 
- entry1
- entry2"?